draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -846,17 +853,14 @@ byte order.  Like an A or AAAA RRSet, the list of addresses represents an
 unordered collection, and clients SHOULD pick addresses to use in a random order.
 An empty list of addresses is invalid.
 
-These parameters MAY be repeated multiple times within a record.
-When receiving such a record, clients SHOULD combine the sets of addresses.
-
 When selecting between IPv4 and IPv6 addresses to use, clients may use an
 approach such as {{!HappyEyeballsV2=RFC8305}}.
-When only "ipv4hint" parameters are present, IPv6-only clients may synthesize
+When only "ipv4hint" is present, IPv6-only clients may synthesize
```

Comment:
Does this restrict us to only one address of each family?

Reply:
Both ipv4hint and ipv6hint are meant to contain an unordered set of addresses.

Looks like it was not included in the diff context, but this appears earlier in the section:

> The wire format for each parameter is a sequence of IP addresses in network
byte order.  Like an A or AAAA RRSet, the list of addresses represents an
unordered collection, and clients SHOULD pick addresses to use in a random order.
An empty list of addresses is invalid.

Reply:
Great, thanks!